src/lib/api/actions.ts

```diff
@@ -65,6 +65,17 @@ export const ActionApi = {
    */
   delete: async (id: Id): Promise<Action> =>
     EntityApi.deleteFn<null, Action>(`${ACTIONS_BASEURL}/${id}`),
+
+  /**
+   * Deletes an action nested under another entity (foreign key relationship).
+   *
+   * @param entityId The ID of the entity under which to delete the action
+   * @param actionId The ID of the action to delete
+   * @returns Promise resolving to the deleted Action object
+   */
+  deleteNested: async (_entityId: Id, _actionId: Id): Promise<Action> => {
```

Comment:
forgive me I just haven't noticed this until now - what are the underscore prefixes on the params? Just acknowledging the api method isn't implemented yet?